src/CSModule.sol

```diff
@@ -1637,9 +1637,12 @@ contract CSModule is
         // The Node Operator is charged for the every removed key. It's motivated by the fact that the DAO should cleanup
         // the queue from the empty batches related to the Node Operator. It's possible to have multiple batches with only one
         // key in it, so it means the DAO should be able to cover removal costs for as much batches as keys removed in this case.
-        uint256 amountToCharge = keyRemovalCharge * keysCount;
-        accounting.chargeFee(nodeOperatorId, amountToCharge);
-        emit KeyRemovalChargeApplied(nodeOperatorId, amountToCharge);
+        uint256 removalFee = keyRemovalCharge;
```

Comment:
If we calculate `amountToCharge` in advance we save space on the stack, I guess.

Reply:
done